### PR TITLE
Added transparent logo and modified page title for seo.

### PR DESCRIPTION
**Pull request by iheartkode**
This works :)

### CODE REVIEW

**Comment by kirs** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source 'https://rails-assets.org' do
   gem 'rails-assets-highcharts', '~> 4.0.4'
 end
 
+
```
Comment (tgxworld) [inline Gemfile]: Could you remove this extra line? Thank you!

Comment (tgxworld) [inline Gemfile]: O just saw it was a commit. Any reason for the extra line?

**Comment by tgxworld** [comment]
Thanks @iheartkode :smile:

**Comment by iheartkode** [comment]
Hmm, That's odd :/

**Comment by iheartkode** [comment]
Just close it please lol

**Comment by tgxworld** [comment]
@iheartkode You can just revert the last commit with `git reset --hard HEAD^` on your branch. After which, `git push origin <branch name>`.

**Comment by tgxworld** [comment]
No worries about pushing up a wrong commit :smile: :smile:

**Comment by tgxworld** [comment]
@iheartkode Thanks for the awesome work on this and I appreciate your effort. I'm going to close this in favor of https://github.com/ruby-bench/ruby-bench-web/pull/68 as we're getting a redesign.

**Pull request closed without merging**